.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,81 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="PyPackageRequirementsInspection" enabled="true" level="WARNING" enabled_by_default="true">
```

Comment:
The 'level' attribute should be capitalized to 'WARNING' for consistency with other attributes in the code.